### DISCUSSION TITLE
does Saleor Elasticsearch option supports product filters?

### DISCUSSION THREAD

**Discussion by zehlon** [saleor/saleor#8539] [Deployment]
Hello, I read in the documents that Saleor search can integrate with Elasticsearch.
in shops with large product listings, it is a good option for product filters to also integrate with Elasticsearch for speed.
is Saleor product filters compatible with Elasticsearch on the frontend or do we need to implement it manually?
thanks

**Comment by karolkielecki** [comment]
@zehlon H,

There is no official integration with Elasticsearch (Saleor 3.0). Also, it's a backend app -- not sure how it can be implemented on the frontend side.
